core/src/main/scala/org/apache/spark/internal/config/Status.scala

```diff
@@ -73,8 +73,8 @@ private[spark] object Status {
 
   val DISK_STORE_DIR_FOR_STATUS =
     ConfigBuilder("spark.appStatusStore.diskStoreDir")
-      .doc("Local directory where to store app status. " +
-           "It's an alternative to the in-memory kv store")
+      .doc("Local directory where to store diagnostic information of SQL executions. " +
```

Comment:
@linhongliu-db  do you plan to add put more data besides the diagnostic info into the local dir?

Reply:
diagnostic should be enough for now.
but in other future, I'd like to see the disk store keep all app status and the in-memory store only be used as a cache layer.

Reply:
> I'd like to see the disk store keep all app status and the in-memory store only be used as a cache layer.

This requires a lot of refactoring...
Let's make it specified for now. Currently, "store app status" is so general and confusing.

Reply:
sure. make sense to me.